extensions/amp-skimlinks/0.1/test/test-tracking.js

```diff
@@ -195,7 +195,7 @@ describes.fakeWin(
             'page-impressions'
           );
 
-          expect(urlVars.data).to.be.a.string;
+          expect(urlVars.data).to.be.a.string();
```

Comment:
I think these are supposed to be `to.be.a('string')`. :)

https://www.chaijs.com/api/bdd/#method_a
https://www.chaijs.com/api/bdd/#method_string